deploy/kubernetes/overlays/examples/cadvisor-args.yaml

```diff
@@ -1,6 +1,9 @@
 # This patch is an example of setting arguments for the cAdvisor container.
 # This set of arguments mirrors what the kubelet currently uses for cAdvisor, 
-# enables only cpu, memory, and diskIO metrics, and shows only container metrics.
+# enables only cpu, memory, diskIO, disk and network metrics, and shows only
+# container metrics.
+#
+# https://github.com/google/cadvisor/blob/master/deploy/kubernetes/overlays/examples/cadvisor-args.yaml
```

Comment:
What is the self-link for?

Reply:
Error, I'm including it in own manifests to find the reference later.